lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -264,6 +299,64 @@ void fetchAccessToken_whenCalledAgainstPassportCri_retrievesAValidAccessToken(
         assertThat(accessToken.getLifetime(), greaterThan(0L));
     }
 
+    @Test
+    @PactTestFor(pactMethod = "invalidAccessTokenRequestReturnsError")
+    void fetchAccessToken_whenCalledAgainstPassportCri_retrievesAnInValidAccessToken(
+            MockServer mockServer) throws URISyntaxException, JOSEException, CriApiException {
+        try {
```

Comment:
Instead of try/catch here you should use `assertThrows()` instead. I think with the current code if `fetchAccessToken()` doesn't throw an exception the test will still pass